pages/data.py

```diff
@@ -22,17 +22,22 @@
 layout = html.Div(
     [
         dcc.Markdown(intro),
-        dcc.Tabs(id="tabs-datatable", value='tab-uuids-datatable', children=[
-            dcc.Tab(label='UUIDs', value='tab-uuids-datatable'),
-            dcc.Tab(label='Trips', value='tab-trips-datatable'),
-            dcc.Tab(label='Demographics', value='tab-demographics-datatable'),
-            dcc.Tab(label='Trajectories', value='tab-trajectories-datatable'),
-        ]),
+        dcc.Tabs(
+            id="tabs-datatable",
+            value='tab-uuids-datatable',
+            children=[
+                dcc.Tab(label='UUIDs', value='tab-uuids-datatable'),
+                dcc.Tab(label='Trips', value='tab-trips-datatable'),
+                dcc.Tab(label='Demographics', value='tab-demographics-datatable'),
+                dcc.Tab(label='Trajectories', value='tab-trajectories-datatable'),
+            ]
+        ),
         html.Div(id='tabs-content'),
         dcc.Store(id='selected-tab', data='tab-uuids-datatable'),  # Store to hold selected tab
-        dcc.Interval(id='interval-load-more', interval=20000, n_intervals=0), # default loading at 10s, can be lowered or hightened based on perf (usual process local is 3s)
+        dcc.Interval(id='interval-load-more', interval=24000, n_intervals=0),  # Interval for loading more data
```

Comment:
Interval increased; ok for now until we implement a better solution that is not as fragile and does not depend on an arbitrary "magic number"